pkg/terraform/stages/compat/stage.go

```diff
@@ -121,16 +116,6 @@ func extractHostAddresses(config *types.InstallConfig, tfstate *terraform.State)
 		if err != nil {
 			logrus.Error(err)
 		}
-	case ovirttypes.Name:
-		bootstrap, err = gatherovirt.BootstrapIP(tfstate)
```

Comment:
I think that both calls should be moved to "ExtractHostAddresses" or we need to add a mechanized for providers to implement this function, similar to "WithCustomDestroy"